contrib_bots/lib/gmail.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import print_function
+#instructions to install googleapiclient can be found in this bot's readme file.
+import base64
```

Comment:
it should be simple. Could we reorganize imports - it looks messy.
From the shortest to the longest.